spec/inject_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'open_array'
+
+describe Array do
+
+  describe '#my_inject' do
+
+    it 'sums numbers' do
+      expect([5,5,5].my_inject{ |sum, next_number | sum + next_number }).to eq(15)
+    end
+
+    it 'sums more numbers' do
+      expect([2,3,4].my_inject{ |sum, next_number | sum + next_number }).to eq(9)
+    end
+
+    it 'subtracts numbers' do
+      expect([5,2].my_inject{ |sum, number| sum - number } ).to eq(3)
+    end
+
+    it 'subtracts more numbers' do
+      expect([2,3,4].my_inject{ |sum, number| sum - number }).to eq(-5)
+    end
+
+    it 'sums with setter' do
+      expect([5,5,5].my_inject(1) { |sum, next_number | sum + next_number }).to eq(16)
```

Comment:
Line is too long. [86/80]<br>Space missing after comma.<br>Space after last block parameter detected.